va/meson.build

```diff
@@ -57,6 +57,13 @@ libva_headers_priv = [
 
 libva_sym = 'libva.syms'
 libva_sym_path = '@0@/@1@'.format(meson.current_source_dir(), libva_sym)
+libva_link_args = '-Wl,-version-script,' + libva_sym_path
```

Comment:
Can we do that vice versa?
```
libva_link_args = []
libva_link_depends = []
if cc.links('', name: '-Wl,--version-script', args: ['-shared', libva_link_args])
  libva_link_args = '-Wl,-version-script,' + libva_sym_path
  libva_link_depends = libva_sym
endif
```

Reply:
libva_link_args needs to be` '-Wl,-version-script,' + libva_sym_path` when checking. I refactored to be almost like your suggestion with that small change